tests/Unit/ProxyFactoryTest.php

```diff
@@ -152,8 +192,10 @@ public function setParent($parent)
 
 class TestProxyDocumentProxy
 {
+    private $title = 'title';
+
     public function getTitle()
     {
-        return 'Hello';
+        return $this->title;
```

Comment:
Methods are not proxied anymore, instead only properties have a proxy. Therefore it is necessary to use a property here as well.